src/data-loaders/LambdaDataLoader.js

```diff
@@ -9,6 +9,17 @@ const getBatchDataResolver = (loaderName, resolver) => {
   return batchLoaders[loaderName];
 };
 
+const getLoaderName = (payload, config) => {
+  const parts = [config.name];
+  if (payload && payload.info && payload.info.parentTypeName) {
```

Comment:
@JimmyHurrah You can use the shorthand

```suggestion
  if (payload?.info?.parentTypeName) {
```

Reply:
Ah yeah, ofc, I was unsure if it was supported, I forgot it got compiled with babel 👍